deployment/Dockerfile.prod

```diff
@@ -0,0 +1,14 @@
+# base_image is define in peerbook/Dockerfile
+# to build base_image run from peerbook:
+# docker build -t base_image .
+FROM base_image
```

Comment:
I don't like having two images especially as one of them can't run. Why not merge this into Dockerfile?

Reply:
Base image is your initial dockerfile, which if not we needed to change in order to have a deployed version. If we merge this into your dockerfile, you and everyone else, will need to start developing differently. The other way to do this, is to utilise docker stages, when developing locally, you will need to remember to build the right stage.